packages/components/Atoms/Link/Link.vue

```diff
@@ -57,8 +57,8 @@ export default {
     if (!isAnchorLink(this.href) && isRelativeUrl(this.href)) {
       this.isNuxtLink = this.rplOptions.nuxt
     }
-    // Set link target for non nuxt-links
-    if (this.target.length === 0) {
+    // Set link target for non nuxt-links, if global option is true
+    if (this.target.length === 0 && this.rplOptions.externalLinksInNewWindow) {
```

Comment:
Do we need update doc for this option? https://github.com/dpc-sdp/ripple/blob/develop/packages/components/Atoms/Global/README.md

Reply:
yep, updated